hooks/synths/useFeeReclaimPeriod.ts

```diff
@@ -0,0 +1,20 @@
+import { useMemo } from 'react';
+import { useRecoilValue } from 'recoil';
+
+import useSynthetixQueries from '@synthetixio/queries';
+import { CurrencyKey } from 'constants/currency';
+import { walletAddressState } from 'store/wallet';
+
+const useFeeReclaimPeriod = (currencyKey: CurrencyKey): number => {
```

Comment:
this could all be in a component. it is basically a hook wrapping a hook without adding too much